website/src/pages/product/automation.tsx

```diff
@@ -0,0 +1,231 @@
+                </h2>
+
+                <div className="row mt-4">
+                    <div className="row justify-content-md-center">
+                        <p className="col-lg-9">
+                            Sourcegraph's automation lets you create campaigns, which are collections of pull requests
+                            and issues across multiple repositories. The following campaign types are built in, and you
+                            can create other ad-hoc campaigns from the UI or use the Sourcegraph extension API for more
+                            advanced campaigns:
+                        </p>
+                        <CampaignsTable
+                            campaigns={[
+                                'Deprecate a Java artifact dependency',
+                                'Deprecate a package.json dependency (npm/yarn)',
+                                'Gradually enforce a new ESLint rule',
+                                'Enforce all files have valid code owners',
+                                'Find-replace across multiple repositories',
+                                'Triage search results',
+                                'Track an existing collection issues and pull requests',
+                            ]}
+                        />
+                    </div>
+                </div>
+
+                <div className="row mt-5">
+                    <div className="row justify-content-md-center">
+                        <p className="col-lg-9">
+                            When you create a campaign, Sourcegraph automatically creates pull requests and issues as
+                            needed with the correct reviewers and context--and keeps them up-to-date. You can track the
+                            progress of the campaign with the burndown chart and see all related activity across all
+                            repositories. All this makes completing the campaign as fast and painless as possible.
+                        </p>
+                    </div>
+                </div>
+
+                <h2 id="easily-define-campaigns-track-progress" className="text-center display-4 mb-3 mt-5">
+                    See it in action
+                </h2>
+                <div className="row justify-content-md-center">
+                    <p className="text-center">
+                        Watch how to use Sourcegraph to automate the deprecation of a JavaScript library across 100+
+                        projects.
+                    </p>
+                </div>
+            </ContentSection>
+            <ContentSection color="white" className="pt-5 pb-3">
+                <span id="customers" />
+                <hr />
+                <CustomerLogosSection className="mt-6 mb-4" />
+            </ContentSection>
+            <ContentSection color="black" className="py-5">
+                <EnterpriseReadySolution />
+            </ContentSection>
+            <Jumbotron
```

Comment:
This final jumbotron looks weird with the "coming soon" — it makes sense at the top of the page, but this makes it look like the demo itself is coming soon.

![image](https://user-images.githubusercontent.com/5589410/63481312-f1ce5a00-c449-11e9-93b8-f9e29729fbb2.png)